### PR TITLE
Kevinliloc

### PR DESCRIPTION
**Pull request by kgliang**
adding 'add new language instruction' on localization.md

### CODE REVIEW

File: Localization.md [hunk 1]
```diff
@@ -23,5 +23,14 @@ We’re using an open localization platform to regularly integrate community fee
 ###Adding new translation suggestions
 * Click ![](./img/suggest.png) to suggest a new translation 
 
+###Adding new language support
+If you want to add new language support for TEE, pleaes follow below steps to request a new language.
```
Comment (AlexRukhlin) [inline Localization.md]: typo
...pleaes...

File: Localization.md [hunk 2]
```diff
@@ -23,5 +23,14 @@ We’re using an open localization platform to regularly integrate community fee
...
+* Go to https://www.transifex.com/microsoft-oss/team-explorer-everywhere/
```
Comment (AlexRukhlin) [inline Localization.md]: Convert to a Markdown link, i.e. `[title](url)`

Comment (AlexRukhlin) [inline Localization.md]: Convert to a Markdown link, i.e. `[title](url)`

File: Localization.md [hunk 3]
```diff
@@ -23,5 +23,14 @@ We’re using an open localization platform to regularly integrate community fee
...
+* Click the Language ![](./img/lang.png) entry under Team Explorer Everywhere in left pane.
+* Then click Edit Languages icon ![](./img/editlang.png)
+* And enter language you want to add ![](./img/addlang.png) and click Apply.
```
Comment (AlexRukhlin) [inline Localization.md]: Enter language...
(No need for "And" here

File: Localization.md [hunk 4]
```diff
@@ -23,5 +23,14 @@ We’re using an open localization platform to regularly integrate community fee
 ###Adding new translation suggestions
 * Click ![](./img/suggest.png) to suggest a new translation 
 
+###Adding new language support
+If you want to add new language support for TEE, pleaes follow below steps to request a new language.
+* Go to https://www.transifex.com/microsoft-oss/team-explorer-everywhere/
+* Click the Language ![](./img/lang.png) entry under Team Explorer Everywhere in left pane.
+* Then click Edit Languages icon ![](./img/editlang.png)
```
Comment (AlexRukhlin) [inline Localization.md]: Add a period (.) at the end of the sentence.

**Comment by AlexRukhlin** [comment]
Please rename all *.PNG files to *.png, i.e. with lower case extensions.

**Comment by kgliang** [comment]
update with new pr. close this one

**Pull request closed without merging**